### ISSUE TITLE
KeyError when adding new segment

### ISSUE THREAD

**Issue by gjost** [denshoproject/ddr-local#285]
```
Request Method: GET
Request URL: http://ddrlocal.local/ui/entity/ddr-testing-40367-1/new-idservice/?model=segment

Traceback (most recent call last):
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/opt/ddr-local/ddrlocal/webui/decorators.py", line 18, in wrapper
    return f(*args, **kwargs)
  File "/opt/ddr-local/ddrlocal/webui/views/decorators.py", line 18, in inner
    return func(request, *args, **kwargs)
  File "/opt/ddr-local/ddrlocal/storage/decorators.py", line 83, in inner
    return func(request, *args, **kwargs)
  File "/opt/ddr-local/ddrlocal/webui/views/entities.py", line 396, in new_idservice
    entity = _create_entity(request, eidentifier, collection, git_name, git_mail)
  File "/opt/ddr-local/ddrlocal/webui/views/entities.py", line 317, in _create_entity
    exit,status = Entity.create(eidentifier, git_name, git_mail)
  File "/opt/ddr-local/ddrlocal/webui/models.py", line 723, in create
    agent=agent,
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.0.2-py3.7.egg/DDR/commands.py", line 51, in wrapper
    return f(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.0.2-py3.7.egg/DDR/commands.py", line 41, in wrapper
    return f(*args, **kwargs)
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.0.2-py3.7.egg/DDR/commands.py", line 464, in entity_create
    git_files.append(eidentifier.path_rel('xml'))
  File "/opt/ddr-local/venv/ddrlocal/lib/python3.7/site-packages/ddr_cmdln-5.0.2-py3.7.egg/DDR/identifier.py", line 1276, in path_rel
    filename = ADDITIONAL_PATHS[self.model][append]
Exception Type: KeyError at /ui/entity/ddr-testing-40367-1/new-idservice/
Exception Value: 'xml'
```

**Comment by gjost** [comment]
This is an issue with `ddr-defs`.  `IDENTIFIERS[4]['files']` is missing an `xml` entry (index 4 is 'segment').

**Comment by gjost** [comment]
Fixed in `ddr-defs` commit `374018e` for package `ddrlocal-master-5.0.3`
